app/feature/feature-insurances/src/main/kotlin/com/hedvig/android/feature/insurances/insurance/InsuranceDestination.kt

```diff
@@ -132,60 +132,58 @@ private fun InsuranceScreen(
       WindowInsets.systemBars.getTop(this).toDp()
     }
     val pullRefreshState = rememberPullRefreshState(
-      refreshing = isLoading,
+      refreshing = isRetrying,
       onRefresh = reload,
       refreshingOffset = PullRefreshDefaults.RefreshingOffset + systemBarInsetTopDp,
     )
     Box {
-      Column(
-        Modifier
-          .pullRefresh(pullRefreshState)
-          .verticalScroll(rememberScrollState())
-          .windowInsetsPadding(WindowInsets.safeDrawing),
-      ) {
-        Row(
-          verticalAlignment = Alignment.CenterVertically,
-          modifier = Modifier
-            .height(64.dp)
-            .fillMaxWidth()
-            .padding(horizontal = 16.dp),
-        ) {
-          Text(
-            text = stringResource(id = R.string.DASHBOARD_SCREEN_TITLE),
-            style = MaterialTheme.typography.titleLarge,
-          )
-        }
+      AnimatedContent(targetState = uiState.isLoading, label = "") { isLoading ->
         Spacer(Modifier.height(16.dp))
```

Comment:
Since this Spacer now is just the child of the AnimatedContent (and not of the column as it was before) I think this does nothing now, so you can remove it.
If we still need this space however, it'd need to be added to the column below here in the isLoading -> false situation or something like that, can't really tell easily without looking at the emitted UI.